admin/google_search_console/class-gsc-table.php

```diff
@@ -333,48 +326,31 @@ private function do_reorder( $a, $b ) {
 	}
 
 	/**
-	 * Modal box
+	 * Checks if premium is loaded, if not the nopremium modal will be shown. Otherwise it will load the orem
```

Comment:
I believe there's a typo here at the end of the comment. Also missing a period at the end.